### PR TITLE
Upgrade protobufer to 3.25.5

### PR DESCRIPTION
**Pull request by slobodanadamovic**
Updating `com.google.protobuf:protobuf-java-util` and `com.google.protobuf:protobuf-java` dependencies to the latest 3.x version.

### CODE REVIEW

File: docs/changelog/113869.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+pr: 113869
+summary: Upgrade protobufer to 3.25.5
+area: HDFS
```
Comment (slobodanadamovic) [inline docs/changelog/113869.yaml:3]: Not sure if this is the best area to apply. I've marked this as HDFS, but actually the protobuf is used in many places.

Comment (slobodanadamovic) [reply docs/changelog/113869.yaml:3]: hm, it turns out it must be one of:

```
Aggregations, Allocation, Analysis, Application, Audit, Authentication, Authorization, Autoscaling, CAT APIs, CCR, CCS, CRUD, Client, Cluster Coordination, Codec, Data streams, DLM, Discovery-Plugins, Distributed, Downsampling, EQL, ES|QL, Engine, Experiences, Extract&Transform, FIPS, Features, Geo, Graph, Health, Highlighting, ILM+SLM, IdentityProvider, Indices APIs, Inference, Infra/CLI, Infra/Circuit Breakers, Infra/Core, Infra/Logging, Infra/Node Lifecycle, Infra/Plugins, Infra/REST API, Infra/Resiliency, Infra/Scripting, Infra/Settings, Infra/Transport API, Infra/Metrics, Ingest, Ingest Node, Java High Level REST Client, Java Low Level REST Client, License, Logs, Machine Learning, Mapping, Monitoring, Network, Packaging, Percolator, Performance, Query Languages, Ranking, Recovery, Reindex, Relevance, Rollup, SQL, Search, Security, Snapshot/Restore, Stats, Store, Suggesters, Task Management, TLS, Transform, TSDB, Vector Search, Watcher
```

Comment (masseyke) [reply docs/changelog/113869.yaml:3]: You probably want this to be Snapshot/Restore since it's for the hdfs snapshot repository.

**Review by masseyke — APPROVED** [review]
LGTM other than the minor changelog problem.

**Review by mark-vieira — APPROVED** [review]

**Comment by slobodanadamovic** [comment]
@elasticmachine update branch

**Comment by slobodanadamovic** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.x|[](https://github.com/elastic/elasticsearch/pull/114117)|
|✅|8.15|[](https://github.com/elastic/elasticsearch/pull/114118)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Comment by slobodanadamovic** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|7.17|[](https://github.com/elastic/elasticsearch/pull/114119)|

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**